MAUI/ProgressBar/migration.md

```diff
@@ -0,0 +1,893 @@
+circularProgressBar.AnimationDuration = 1500;
+circularProgressBar.IndeterminateAnimationDuration = 2000;
+circularProgressBar.IsIndeterminate = false;
+circularProgressBar.ProgressFill = Colors.Red;
+circularProgressBar.TrackFill = Colors.Violet;
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("00bdaf"), Value = 0 });
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("2f7ecc"), Value = 50 });
+this.Content = circularProgressBar;
+
+...
+
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+</tr>
+</table>
+
+
+## SfLinearProgressBar
+
+<table> 
+<tr>
+<th>Xamarin ProgressBar</th>
+<th>.NET MAUI ProgressBar</th>
+<th>Description</th></tr>
+<tr>
+<td>{{'[SecondaryProgress](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfLinearProgressBar.html#Syncfusion_XForms_ProgressBar_SfLinearProgressBar_SecondaryProgress)'| markdownify }}</td>
+<td>{{'SecondaryProgress'| markdownify }}</td>
+<td>Gets or sets the secondary progress value for the SfLinearProgressBar.</td></tr>
+<tr>
+<td>{{'[SecondaryProgressColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfLinearProgressBar.html#Syncfusion_XForms_ProgressBar_SfLinearProgressBar_SecondaryProgressColor)'| markdownify }}</td>
+<td>{{'SecondaryProgressFill'| markdownify }}</td>
+<td>Gets or sets the brush that paints the interior area of the secondary progress.</td></tr>
+<tr>
+<td>{{'[TrackHeight](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfLinearProgressBar.html#Syncfusion_XForms_ProgressBar_SfLinearProgressBar_TrackHeight)'| markdownify }}</td>
+<td>Divided into {{'TrackHeight'| markdownify }}, {{'ProgressHeight'| markdownify }} and {{'SecondaryProgressHeight'| markdownify }}</td>
+<td>Gets or sets a value to determine the height of track, progress and secondary progress.</td></tr>
+<tr>
+<td>{{'[CornerRadius](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfLinearProgressBar.html#Syncfusion_XForms_ProgressBar_SfLinearProgressBar_CornerRadius)'| markdownify }}</td>
+<td>Divided into {{'TrackCornerRadius'| markdownify }}, {{'ProgressCornerRadius'| markdownify }} and {{'SecondaryProgressCornerRadius'| markdownify }}</td>
+<td>Gets or sets a value to determine the corner radius of the track, progress, and secondary progress.</td></tr>
+<tr>
+<td>{{'[SecondaryAnimationDuration](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfLinearProgressBar.html#Syncfusion_XForms_ProgressBar_SfLinearProgressBar_SecondaryAnimationDuration)'| markdownify }}</td>
+<td>{{'SecondaryAnimationDuration'| markdownify }}</td>
+<td>Gets or sets a value that specifies the secondary progress animation duration in milliseconds.</td></tr>
+
+</table> 
+
+The following code example explains how to initialize the Xamarin SfLinearProgressBar and the .NET MAUI SfLinearProgressBar along with its properties.
```

Comment:
The following code example explains how to initialize the Xamarin SfLinearProgressBar and the .NET MAUI SfLinearProgressBar along with its properties: